src/main/java/io/apimatic/core/ResponseHandler.java

```diff
@@ -171,23 +210,49 @@ private void validateResponse(Context httpContext) throws ExceptionType {
             globalErrorCases.get(errorCode).throwException(httpContext);
         }
 
-        if ((statusCode < 200) || (statusCode > 208)) {
+        if ((statusCode < SUCCESS_CODE) || (statusCode > ALREADY_REPORTED)) {
```

Comment:
Since this is a range where we consider the API call was successful so these constants should be renamed to MIN_SUCCESS_CODE and MAX_SUCCESS_CODE,

Reply:
resolved